main.pm

```diff
@@ -206,7 +206,8 @@ sub lxdestep_is_applicable() {
 }
 
 sub snapper_is_applicable() {
-    return (check_var("FILESYSTEM", "btrfs") && get_var("HDDSIZEGB") > 10);
+    my $fs = get_var("FILESYSTEM") || 'btrfs';
```

Comment:
get_var takes second optional parameter indicating the default if var is not set. So `get_var("FILESYSTEM", btrfs);`